### ISSUE TITLE
Unknown theme 'Default'

### ISSUE THREAD

**Issue by anntzer** [sharkdp/bat#244]
After installing a theme (https://github.com/colinta/zenburn), running `bat cache -i` appears to drop the default themes (`bat --list-themes` only prints `zenburn`).  Running `bat` on a file (without setting the theme to zenburn) fails:
```
[bat warning]: Unknown theme 'Default', using default.
thread 'main' panicked at 'no entry found for key', libcore/option.rs:960:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
```

**Comment by sharkdp** [comment]
Can you please make sure that you are running the latest version of `bat`?

This should have been resolved in 0.6. 

Also, `zenburn` is included by default, now.

**Comment by anntzer** [comment]
Sorry, I was on 0.5.  zenburn works directly on 0.6, as you mentioned.  I didn't try with a third-party theme but I saw that this was directly addressed by some other isses, so I assume they do work.
Thanks!
